library/BerlinPipeline/src/Juvix/BerlinPipeline/Meta.hs

```diff
@@ -17,26 +23,18 @@ data T = Meta
   deriving (Eq, Show)
 
 data Success a = Success
-  { meta :: T,
-    result :: a
+  { sMeta :: T,
+    sResult :: a
```

Comment:
I think we should keep the same names and use `DuplicateRecordFields` pragma, that way the interface becomes clearer.